File: API/Models/Employee.cs
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
    [Table("Employees")]
    public class Employee
    {
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Surname { get; set; }
        public string Patronymic { get; set; }

        public ICollection<Item> Items { get; set; }

        public Employee()
        {
            Items = new Collection<Item>();
        }
    }
}

File: SPA/src/app/interfaces/employee.ts
import { Item } from './item';

export class Employee {
    id: number;
    name: string;
    surname: string;
    patronymic: string;
    items: Item[];

    constructor() {
        this.id = 0;
        this.name = '';
        this.surname = '';
        this.patronymic = '';
        this.items = [];
    }
}